src/models/types/index.d.ts

```diff
@@ -19,6 +19,6 @@ declare global {
 			| number
 			| boolean
 			| Record<string, string | number | boolean>
-		>;
+		> | any;
```

Comment:
No anys! Also, is this necessary? Would `Record<string, string | number | boolean | Record<string, string | number | boolean>>` cover most events?

Reply:
So I was not able to get this working with that, it seems the issue is that all these objects are expected the Record to have a string key value but in the case of the "event" that the browser_mod is expecting to receive:
```
{
  action: 'fire-dom-event', 
  browser_mod: { 
    data: { 
      content: "Test",
      deviceID: "THIS",
      title: "test"
    }
    service: "browser_mod.popup"
  }
}
```

seems to be just the dictionary without a set key value. I fixed it to also extend to | IAction but not sure that is the best way

Reply:
Thinking about the format of this event:

```typescript
const event = {
    detail: {
        browser_mod: {
            data: { 
              content: "Test",
              deviceID: "THIS",
              title: "test"
            }
            service: "browser_mod.popup"
        }
    }
}
```

It looks like the record value can be a string, a record of value string, or a record of a record of value string. Chaining it one level deeper would fix the issue but it's also messy. What we could do is an es-lint exception for the Event definition since we do want this to be a `Record<string, any>` as it's user defined and can really anything.

```yaml
interface Event {
    // eslint-disable-next-line
    detail?: Record<string, any>
}
```

Reply:
I've made some changes for a small fix (and README updates), so you'll need to pull the latest copy of your branch before making the changes.

Reply:
This makes sense, should be updated feel free to give it a once over when you have time. Thanks.

Reply:
It all looks good to me! I'll go ahead and merge this into my dev branch, make an alpha build for it to make sure that everyone works okay, and add it to the README. Thanks for all your hard work! It's nice to be able to collaborate on this.